pykern/pkio.py

```diff
@@ -18,11 +18,26 @@
 import re
 import shutil
 import six
+import tempfile
 
 #: used during unit testing see ``pykern.pkunit.save_chdir``
 pkunit_prefix = None
 
 
+def atomic_write(filename, content, mode='w'):
+    """Writes to file atomically
+
+    Args:
+        filename (str or py.path.Local): File to write to
+    """
+    with tempfile.NamedTemporaryFile(
```

Comment:
I think we should just name the file with a suffix, e.g. '.pkio-tmp' and not use a library.